<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuncConfigDialog</class>
 <widget class="QDialog" name="FuncConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>功能配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>销贷锁车功能</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="lockFuncCheckBox">
           <property name="text">
            <string>开启销贷锁车</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="lockVehicleButton">
           <property name="text">
            <string>锁车配置表</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>提示：ECU锁车都是在下次上电后才生效！</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>380</height>
          </size>
         </property>
         <property name="title">
          <string>TEA算法校验</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="activeRadioButton">
              <property name="text">
               <string>激活密文</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="handshakeRadioButton">
              <property name="text">
               <string> 握手密文</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="clearInputButton">
              <property name="text">
               <string>清空输入</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="decryptButton">
              <property name="text">
               <string>解密</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>输入区</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="3" column="0">
              <widget class="QLabel" name="ciphertext3Label">
               <property name="text">
                <string>密文3：0x</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="2">
              <widget class="QLineEdit" name="encryptSeedLineEdit">
               <property name="text">
                <string>A1 44 78 06 8F 9C F4 57</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="ciphertext1LineEdit">
               <property name="text">
                <string>01 E0 D1 5F 31 EF B3 44</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="ciphertext2Label">
               <property name="text">
                <string>密文2：0x</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="encryptKeyLabel">
               <property name="text">
                <string>加密密钥：0x</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="2">
              <widget class="QLineEdit" name="ciphertext3LineEdit">
               <property name="text">
                <string>03 87 39 00 00 00 00 00</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="encryptSeedlabel">
               <property name="text">
                <string>SEED： 0x</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="encryptKeyLineEdit">
               <property name="text">
                <string>32 B1 62 CD 72 9F 6E 13 5F AE 2E 12 12 74 7A 3E</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="ciphertext1Label">
               <property name="text">
                <string>密文1：0x</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QLineEdit" name="ciphertext2LineEdit">
               <property name="text">
                <string>02 DC 49 91 22 C4 6D CE</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>  输出区</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="0">
              <widget class="QLabel" name="plaintext2Label">
               <property name="text">
                <string>明文2：0x</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="plaintext2LineEdit"/>
             </item>
             <item row="4" column="0" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="gpsidLabel">
                 <property name="text">
                  <string>GPSID：0x</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="gpsidLineEdit"/>
               </item>
               <item>
                <widget class="QLabel" name="decryptSeedLabel">
                 <property name="text">
                  <string>SEED：0x</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="decryptSeedLineEdit"/>
               </item>
              </layout>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QLineEdit" name="plaintext3LineEdit"/>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QLabel" name="manualKeyLabel">
               <property name="text">
                <string>手动密钥：0x</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLineEdit" name="plaintext1LineEdit"/>
             </item>
             <item row="5" column="2">
              <widget class="QLineEdit" name="manualKeyLineEdit"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="plaintext1Label">
               <property name="text">
                <string>明文1：0x</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="plaintext3Label">
               <property name="text">
                <string>明文3：0x</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>疲劳驾驶功能</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="tiredFuncCheckBox">
         <property name="text">
          <string>开启疲劳驾驶</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>最大连续运行时间：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>秒</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>提 前 预 警 时间：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>秒</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>最小连续休息时间：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>秒</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="maxRunLineEdit">
         <property name="text">
          <string>14400</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="minRestLineEdit">
         <property name="text">
          <string>1200</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="preWarnLineEdit">
         <property name="text">
          <string>1800</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
